user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -951,11 +961,190 @@ end
+--! "SELECT functional_group_id FROM app_group WHERE application_id=\\\"0000001\\\""
 --! @param exp_result contains data for comparing data from DB
 --! @return Returns false if expected data are not equal with DB data, otherwise returns true.
 function commonFunctions:is_db_contains(db_path, sql_query, exp_result)
   local column_db = commonFunctions:get_data_policy_sql(db_path, sql_query)
   return commonFunctions:is_table_equal(column_db, exp_result)
 end
 
+--------------------------------------------------------------------------------------------
+--20. Function checks PTU sequence need to call when PTU is triggered before call of function – it should just finalize PTU
+-- Checkes PTU HTTP flow.
+---------------------------------------------------------------------------------------------
+--! @brief Checks PTU HTTP flow sequence
+--! @param ptu_path path to PTU file
+--! @param ptu_name contains name of PTU file
+function commonFunctions:check_ptu_sequence_partly(self, ptu_path, ptu_name)
+  check_file_existing(ptu_path)
+  local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+    {
+      requestType = "HTTP",
+      fileName = ptu_name,
+    },ptu_path)
+  EXPECT_HMICALL("BasicCommunication.SystemRequest"):Times(0)
+  EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+  :Timeout(10000)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate")
+      :ValidIf(function(exp,data)
+        if
+          exp.occurences == 1 and
+          data.params.status == "UPDATING" then
+          return true
+        elseif
+          exp.occurences == 2 and
+          data.params.status == "UP_TO_DATE" then
+          return true
+        else
+          return false
+        end
+      end):Times(2)
+end
+
+--------------------------------------------------------------------------------------------
+--21. Function checks PTU sequence fully need to call when PTU is UP_TO_DATE, and PTU was triggerd. 
+---------------------------------------------------------------------------------------------
+--! @brief Checks PTU HTTP flow fully sequence
+--! @param ptu_path path to PTU file
+--! @param ptu_name contains name of PTU file
+function commonFunctions:check_ptu_sequence_fully(self, ptu_path, ptu_name)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status="UPDATE_NEEDED"})
+  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"}) 
+  commonFunctions:check_ptu_sequence_partly(self, ptu_path, ptu_name)
+end
+
+--------------------------------------------------------------------------------------------
+--Function fills request parameters for subscribe unsubscrive vehicle data
+---------------------------------------------------------------------------------------------
+--! @brief Fills table with parameter for subscribe unsubscribe vehicle data request 
+--! @param vehicle_data contains name of parameters
+--! @return Returns table with request parameters
+local function fill_request_parameters(vehicle_data)
```

Comment:
@dtrunov looks like t fill all request parameters with true. It should be clarifies in description or in func name